pylint/checkers/utils.py

```diff
@@ -1164,11 +1164,15 @@ def supports_getitem(value: nodes.NodeNG, node: nodes.NodeNG) -> bool:
     return _supports_protocol(value, _supports_getitem_protocol)
 
 
-def supports_setitem(value: nodes.NodeNG, *_: Any) -> bool:
+def supports_setitem(
+    value: nodes.NodeNG, node: nodes.NodeNG  # pylint: disable=unused-argument
+) -> bool:
```

Comment:
I don't really like adding `pylint: disable` here. Better to do `_: nodes.NodeNG` instead.

Reply:
Done!